queue_services/account-mailer/src/account_mailer/config.py

```diff
@@ -116,6 +116,31 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     TEMPLATE_PATH = os.getenv('TEMPLATE_PATH', 'src/account_mailer/email_templates')
 
+    # JWT_OIDC Settings
+    JWT_OIDC_WELL_KNOWN_CONFIG = os.getenv('JWT_OIDC_WELL_KNOWN_CONFIG')
```

Comment:
Please check if these JWT_ variables are needed, as they are used in jwt library, and needed only when parsing the token (API)

Reply:
will clean this up